.github/workflows/lth.yml

```diff
@@ -1,10 +1,16 @@
 name: Liquibase Test Harness
 
 on:
+  workflow_dispatch:
   pull_request:
   push:
     branches:
       - main
+      -
+# ensure only one run or job within the group runs at a time.
+  concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
```

Comment:
```
    strategy:
      max-parallel: 1
```
was working fine. I think the issues is when a matrix jobs fails and the schema is not correctly removed.

Reply:
we do have this step here to ensure removal of the test schema even if one or more matrix jobs fail.  : https://github.com/liquibase/liquibase-databricks/blob/main/.github/workflows/lth.yml#L73 destroying the schema

Reply:
right, I was looking at a way to destroy the resources if creation fails as the resources already exists, but TF doesnt have the context and therefore shows "No changes" : tested here: https://github.com/liquibase/liquibase-databricks/actions/runs/10081580514/workflow#L52. I have set cancel-in-progress: false for the same reason.